.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```
@@ -0,0 +1,48 @@
+name: "🐛 Bug Report"
+description: Create a new ticket for a bug.
+title: "🐛 [BUG] - <title>"
```

```suggestion
name: "🐛 Bug Report"
description: Create a new ticket for a bug.
title: "[BUG] - <title>"
```

Assuming that the title is what's shown on the Issues page.